src/test/regress/expected/dynamic_table.out

```diff
@@ -0,0 +1,326 @@
+-- start_matchsubs
+-- m/ERROR:  can not drop a internal task "gp_dynamic_table_refresh_.*/
+-- s/ERROR:  can not drop a internal task "gp_dynamic_table_refresh_.*/ERROR:  can not drop a internal task "gp_dynamic_table_refresh_xxx"/g
+-- m/WARNING:  relation of oid "\d+" is not dynamic table/
+-- s/WARNING:  relation of oid "\d+" is not dynamic table/WARNING:  relation of oid "XXX" is not dynamic table/g
+-- end_matchsubs
+CREATE SCHEMA dynamic_table_schema;
+SET search_path TO dynamic_table_schema;
+SET optimizer = OFF;
+CREATE TABLE t1(a int, b int, c int) DISTRIBUTED BY (b);
+INSERT INTO t1 SELECT i, i + 1, i + 2 FROM GENERATE_SERIES(1, 10) i;
+INSERT INTO t1 SELECT i, i + 1, i + 2 FROM GENERATE_SERIES(1, 5) i;
+ANALYZE t1;
+CREATE DYNAMIC TABLE dt0 SCHEDULE '5 * * * *' AS
+  SELECT a, b, sum(c) FROM t1 GROUP BY a, b DISTRIBUTED BY(b);
```

Comment:
support distinct ?

Reply:
Off course,
> The Query allows any valid SELECT SQL of Materialized Views: from single or multiple relations, base tables, materialized views, and dynamic tables as well, joins, subquery, aggregation, group by and etc.

Reply:
That's AQUMV part, not Dynamic Table feature itself. 
Users can create Dynamic Table with any SQLs as MV, and if it could be used to answer queries is limited to AQUMV.
See PR description.